### PR TITLE
ANC - final fixes

### LINKED ISSUE #413: ANC - Multiple referral options [12h]
At current state. we can refer to single facility, while there can be 3 different option:

- Hospital (or HC for CHW)
- Mental health specialist
- HIV program

We want to allow reffering to multiple facilities. Each option with it's follow up questions.

### PR DESCRIPTION
**Pull request by anvmn**
#413
#375 
#337 
#252 
#354 
#387 
#368 
#268 
#340

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-09-18 12-15

**Comment by anvmn** [comment]
Selection_1099

**Comment by anvmn** [comment]
### Medication:

Selection_1100


### Hospitalization:

Selection_1101

**Comment by anvmn** [comment]
@AronNovak please review

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -5207,10 +5208,16 @@ translationSet trans =
...
-            }
+        HypertensionRecommendedTreatmentHeader isChronic ->
+            if isChronic then
+                { english = "This patient shows signs of Chronic hypertension"
+                , kinyarwanda = Just "Uyu murwayi agaragaza ibimenyetso by'indwara y'umuvuduko w'amaraso imaze igihe kirekire"
+                }
+
+            else
+                { english = "This patient shows signs of Pregnancy-Induced hypertension"
+                , kinyarwanda = Nothing
```
Comment (AronNovak) [inline client/src/elm/Translate.elm:5227]: will we have it later? Just asking because of "final fixes" in the PR title

**Review by AronNovak — APPROVED** [review]

File: client/src/elm/Translate.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Translate.elm:5227]: @AronNovak 
Translations are never complete. We'll add them when they become available.

**Pull request merged**